src/modules/billing/routes/invoices.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const Joi = require('@hapi/joi');
+
+const controller = require('../controllers/invoices');
+const { ROLES: { billing } } = require('../../../lib/roles');
+
+exports.patchInvoice = {
+  method: 'PATCH',
+  path: '/water/1.0/billing/invoices/{invoiceId}',
+  handler: controller.patchInvoice,
+  config: {
+    validate: {
+      params: Joi.object({
+        invoiceId: Joi.string().guid().required()
+      }),
+      payload: Joi.object({
+        isFlaggedForRebilling: Joi.boolean().optional()
```

Comment:
if isFlaggedForRebilling is the only property you can update then it should be required

Reply:
I did think that as well, but everything else is done in a much more generic way, but maybe that's because I'm reusing a lot of helper methods. I'm happy to change this :) Thank you